tests/bluetooth/controller/ctrl_ull_dletime/CMakeLists.txt

```
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+if (NOT BOARD STREQUAL unit_testing)
+	message(FATAL_ERROR "This project can only be used for unit-testing.")
```

Just to help the user a bit here, so when he uses a wrong board, then please print the correct name with an underscore, and maybe a bit more to really make it clear.
```suggestion
  message(FATAL_ERROR "This project can only be used with `-DBOARD=unit_testing`")
```